meta-ivi/recipes-extended/node-startup-controller/node-startup-controller_1.0.3.bb

```diff
@@ -45,14 +42,3 @@ FILES_${PN} += "\
 FILES_${PN}-dbg += "\
     ${libdir}/${BPN}-1/.debug/*ler \
     "
-
-FILES_${PN}-nsm-dummy = "\
```

Comment:
was the nsm dummy package removed upstream? Is there any other reason it was removed?

Reply:
It was only useful before NodeStateManager was done I guess, and maybe if you want to run component testing. But there's really no reason why it should be shipped on target.

It was also disabled from being built by default after NSM was done, see https://github.com/GENIVI/node-startup-controller/commit/fba1b26575c75390be28a8a020953f030114df87.

Reply:
Hm, I see your point. So let's get rid of it then.... I'm accepting this PR.